aten/src/ATen/native/Activation.cpp

```diff
@@ -850,6 +855,23 @@ std::tuple<Tensor&, Tensor&> log_sigmoid_forward_out_cpu(const Tensor& input, Te
   return std::forward_as_tuple(result, buffer);
 }
 
+std::tuple<Tensor&, Tensor&> log_sigmoid_forward_out_cuda(const Tensor& input, Tensor& result, Tensor& buffer) {
```

Comment:
similar to glu, there's no need to do stubs here, log_sigmoid*cuda can go directly to Activation.cu